spec/frameSpec.js

```diff
@@ -0,0 +1,71 @@
+//
```

Comment:
Remove? Or do not commit.
If it was committed at some point, you can always go back through the commit history to find it again.